### PR TITLE
New rule for scrollbar structure

### PR DESCRIPTION
**Pull request by karanbirsingh**
#### Describe the change
(Please provide an overview of the change in this PR)

This PR adds a rule that checks whether Control View scrollbars adhere to the structure given here: https://docs.microsoft.com/en-us/windows/desktop/WinAuto/uiauto-supportscrollbarcontroltype

It is partly in response to issue #274 - when a complex structure like a tab has children controls like a scrollbar, the scrollbar's children can be checked using this rule instead of checking the full tree at the tab-level.

### CODE REVIEW

File: src/AccessibilityInsights.Rules/Library/Structure/ControlView/Scrollbar.cs [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+        public override EvaluationCode Evaluate(IA11yElement e)
+        {
+            if (e == null) throw new ArgumentException(nameof(e));
+
+            return ControlView.ScrollbarStructure.Matches(e) ? EvaluationCode.Pass : EvaluationCode.Note;
+        }
+
+        protected override Condition CreateCondition()
+        {
+            return ScrollBar;
```
Comment (neocln) [inline src/AccessibilityInsights.Rules/Library/Structure/ControlView/Scrollbar.cs]: this rule only run when the ScrollBar control is marked with IsControlElementProperty true value. 
Please make sure that condition is added.

File: src/AccessibilityInsights.RulesTest/Library/Structure/ControlView/ScrollbarTest.cs [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+using Moq;
+using System.Collections.Generic;
+using System.Linq;
+using EvaluationCode = Axe.Windows.Rules.EvaluationCode;
+
+namespace Axe.Windows.RulesTest.Library
+{
+    [TestClass]
+    public class ControlViewScrollbarStructureTest
+    {
```
Comment (neocln) [inline src/AccessibilityInsights.RulesTest/Library/Structure/ControlView/ScrollbarTest.cs]: Test cases should be updated based on the other comment on ScrollBar.cs file.

File: src/AccessibilityInsights.Rules/Library/Structure/ControlView/Scrollbar.cs (hunk 1 above)
Comment (karanbirsingh) [reply src/AccessibilityInsights.Rules/Library/Structure/ControlView/Scrollbar.cs]: I think this is abstracted inside the `ScrollBar` condition:
https://github.com/Microsoft/accessibility-insights-windows/blob/e455215a219a975d9ac05be7d25f37719a6e613a/src/AccessibilityInsights.Rules/PropertyConditions/ControlType.cs#L33

Comment (karanbirsingh) [reply src/AccessibilityInsights.Rules/Library/Structure/ControlView/Scrollbar.cs]: I mistakenly thought the ScrollBar condition handled this. I will check with @RobGallo - it seems like none of the rules consider this property so we may need to fix it separately:
> Another such failing is that the children are not actually checked to make sure they have the IsContentElement or IsControlElement properties set to true. This failing is common throughout these rules. And again, there is no intent to remedy that as part of this change.
from #281

Comment (karanbirsingh) [reply src/AccessibilityInsights.Rules/Library/Structure/ControlView/Scrollbar.cs]: tracking in #282

**Pull request merged**